N/NamedTupleTools/Compat.toml

```diff
@@ -1,2 +1,2 @@
-[0]
+["0-0.7.0"]
 julia = "1"
```

Comment:
Can you keep this restriction by adding 
```toml
[compat]
julia = "1"
```
to your `Project.toml` file?

Reply:
I put this instead -- it better matches my intent.  does that work well?
```
[compat]
julia = "≥ 1.0"
```

Reply:
But `"1"` means `1.0`, `1.1`, `1.2` etc (probably what you intended), while with `">=1"` you claim to be compatible with julia 2, 3 etc which probably is false.

Reply:
I thought `packagename >= 1.0` meant it would hold for 1.0, 1.1 .. 1.max and the subnumbers but not hold for the rollover to 2.0.  I misunderstood (someone once wrote it up that way).  Corrected.

Reply:
No, see https://julialang.github.io/Pkg.jl/v1/compatibility/.
You also have to retrigger the bot and this PR will be updated with your changes.

Reply:
do I need that caret "^0" ?

Reply:
No, they are default.